diff -up trunk/CMakeLists.txt.omv~ trunk/CMakeLists.txt
--- trunk/CMakeLists.txt.omv~	2021-12-23 21:26:24.279579742 +0100
+++ trunk/CMakeLists.txt	2021-12-23 21:27:33.719744695 +0100
@@ -25,17 +25,6 @@ endif(UNIX)
 
 #set(CMAKE_RELATIVE_PATHS ON CACHE STRING "avoid absolute paths" FORCE)
 
-# Clear out the built-in C++ compiler and link flags for each of the 
-# unused configurations.
-set(CMAKE_CXX_FLAGS_MINSIZEREL "" CACHE INTERNAL "Unused")
-set(CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL "" CACHE INTERNAL "Unused")
-set(CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL "" CACHE INTERNAL "Unused")
-set(CMAKE_EXE_LINKER_FLAGS_MINSIZEREL "" CACHE INTERNAL "Unused")
-set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "" CACHE INTERNAL "Unused")
-set(CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO "" CACHE INTERNAL "Unused")
-set(CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO "" CACHE INTERNAL "Unused")
-set(CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO "" CACHE INTERNAL "Unused")
-
 set(CMAKE_OSX_ARCHITECTURES i386 ppc x86_64 CACHE STRING "change to needed architecture for a smaller library" FORCE)
 
 PROJECT(portmidi)
diff -up trunk/pm_dylib/CMakeLists.txt.omv~ trunk/pm_dylib/CMakeLists.txt
--- trunk/pm_dylib/CMakeLists.txt.omv~	2021-12-23 21:28:23.191867857 +0100
+++ trunk/pm_dylib/CMakeLists.txt	2021-12-23 21:30:07.078139710 +0100
@@ -117,7 +117,7 @@ prepend_path(SHARED_PATHS ../pm_common/
 list(APPEND LIBSRC ${SHARED_PATHS})
 
 add_library(portmidi-dynamic SHARED ${LIBSRC})
-set_target_properties(portmidi-dynamic PROPERTIES OUTPUT_NAME "portmidi")
+set_target_properties(portmidi-dynamic PROPERTIES OUTPUT_NAME "portmidi" SOVERSION 0)
 target_link_libraries(portmidi-dynamic ${PM_NEEDED_LIBS})
 
 # install the libraries (Linux and Mac OS X command line)
