diff -rupN portmidi-2.0.8.old/CMakeLists.txt portmidi-2.0.8/CMakeLists.txt
--- portmidi-2.0.8.old/CMakeLists.txt	2025-12-02 17:11:35.000000000 +0100
+++ portmidi-2.0.8/CMakeLists.txt	2026-05-01 20:44:51.096527679 +0200
@@ -12,7 +12,7 @@ cmake_policy(SET CMP0091 NEW) # enables
 # Versions are now in the form x.y.z
 # Changed 1.0 to 2.0 because API is extended with virtual ports:
 set(SOVERSION "2")
-set(VERSION "2.0.7")
+set(VERSION "2.0.8")
 
 project(portmidi VERSION "${VERSION}"
                  DESCRIPTION "Cross-Platform MIDI IO")
